docs/content/doc/help/faq.en-us.md

```diff
@@ -118,6 +118,12 @@ The correct path for the template(s) will be relative to the `CustomPath`
 2. If you are still unable to find a path, the default can be [calculated above](#where-does-gitea-store-what-file)
 3. Once you have figured out the correct custom path, you can refer to the [customizing Gitea]({{< relref "doc/administration/customizing-gitea.en-us.md" >}}) page to add your template to the correct location.
 
+## Does Gitea have a "GitHub/GitLab pages" feature?
+
+No, Gitea does not ship with that functionality and most likely never will.
+
+However, there is a battle-tested third-party service called [pages server](https://codeberg.org/Codeberg/pages-server) that can be run easily in combination with Gitea.
```

Comment:
Caddy is another resolution

Reply:
@lunny do you have a link to point to?

Reply:
I don't think we have an official one. But heard many users do like that. i.e. https://github.com/42wim/caddy-gitea is a caddy plugin